lms/djangoapps/mobile_api/users/views.py

```diff
@@ -239,6 +239,8 @@ class UserCourseEnrollmentsList(generics.ListAPIView):
             the course.
           * video_outline: The URI to get the list of all videos that the user
             can access in the course.
+          * discussion_url: None if discussion is not enabled for course else
+            the URI to get discussions against the course
```

Comment:
Change to:  `The URI to access data for course discussions if it is enabled, otherwise null.`